01-front-end-basics/jquery-dom-manipulation/27-28/solutions.md

```diff
@@ -0,0 +1,99 @@
+# jQuery & DOM Exercises
+
+#### Nature Drawing 101: Draw a Tree!
+* How many children does `header` have?
+    - just one: the `ul` element
+* Name a direct child of the `p` element.
+    - the `a` elment
+* Name a direct parent of the `p` element
+    - the `section` element
+* What is the parent of the `html` element?
+    - Trick question! It turns out that `html` belongs to the `document` object, which lives inside your browser (not your html)!
+
+#### CSS Selector Challenge
+1.
+* \#fancy
+* p#fancy
+* div p#fancy
+* .page #fancy
+* .page p#fancy
+* div.page p#fancy
+
+2.
+* div .small
+* div a.small
+* .small
+* div.page a.small
+
+3.
+* p a
+* div.page p a
+* .page p a
+* div p a
+
+4.
+* .mad-favs li
+* ul.mad-favs li
+* div.page ul.mad-favs li
+* .page .mad-favs li
+
+5.
+* .sparkles
+* p.sparkles
+* div p.sparkles
+* div.page p.sparkles
+
+6.
+* p a.small
+* p .small
+* div p .small
+* div.page p a.small
+
+#### Indiana Jones And the Temple of DOM
```

Comment:
Unfortunately this has proven to be a very brittle lab. These are now broken.

I'm inclined to say this should be abandoned. But I still love the "theme". A student riffed on the gif and came up with the pun. Love it.

Also, when did the GA site become so spammy!? There were 3 pop-ups in the time that I was checking these.